src/main/java/oxgame/app/VictoryChecker.java

```diff
@@ -0,0 +1,149 @@
+package oxgame.app;
+
+public class VictoryChecker {
+
+    public static boolean check(Coordinates coordinates, Board board, int winningStroke) {
+        return verifyHorizontal(coordinates,board,winningStroke) ||
+                verifyVertical(coordinates,board,winningStroke) ||
+                verifySlash(coordinates,board,winningStroke);
+    }
+    /**
+     * Verify if there is a win on HORIZONTAL.
+     * */
+    private static boolean verifyHorizontal(Coordinates coordinates, Board board, int winningStroke){
+        Symbol currentSymbol = board.getSymbol(coordinates);
+        int counter = 1;
+        counter = horizontalRight(coordinates, board, currentSymbol, counter);
+        counter = horizontalLeft(coordinates, board, currentSymbol, counter);
+        if (counter>=winningStroke) {
+            return true;
+        }
+        return false;
+    }
+
+    private static int horizontalRight(Coordinates coordinates, Board board, Symbol currentSymbol, int counter) {
+        int condition = 1;
+        int coordinatesValue = coordinates.getValue();
+        while ( (currentSymbol.equals(board.getSymbol(coordinatesValue + condition) ))
```

Comment:
For such long conditions its nice to create appropriately named wrapper method

Reply:
Done